src/tools/rbd_mirror/instance_watcher/Types.h

```diff
@@ -44,7 +44,7 @@ struct ImagePayloadBase : public PayloadBase {
   std::string peer_mirror_uuid;
   std::string peer_image_id;
 
-  ImagePayloadBase() : PayloadBase() {
+  ImagePayloadBase() : PayloadBase(), global_image_id{""}, peer_mirror_uuid{""}, peer_image_id{""} {
```

Comment:
@joscollin It looks like I can't reproduce with my compiler. Could you tell your version?
Wont't it fix the warning if you do on the contrary, i.e. remove unnecessary PayloadBase() from the initialization?
```
ImagePayloadBase() {
}
```

Reply:
#18504 silences this warning just to close the loop here.